### ISSUE TITLE
Token or server host is empty, will not start report thread!

### ISSUE THREAD

**Issue by scarletbright** [ZingerLittleBee/server_bee-backend#85]
Getting these on three different servers:

```
[2024-03-26 17:08:56 main INFO] Actix runtime found; starting in Actix runtime
[2024-03-26 17:08:56 main WARN] Token or server host is empty, will not start report thread!
```

then this while trying to connect (the port is opened):
```
[2024-03-26 18:13:41 actix-rt|system:0|arbiter:1 ERROR] stream error: request parse error: invalid Header provided
[2024-03-26 18:13:42 actix-rt|system:0|arbiter:0 ERROR] stream error: request parse error: invalid Header provided
```

**Comment by ZingerLittleBee** [comment]
- `Token or server host is empty, will not start report thread!` no need to care, this is waring. and will remove in latest version.
- `[2024-03-26 18:13:41 actix-rt|system:0|arbiter:1 ERROR] stream error: request parse error: invalid Header provided.` need more info, like version, more logs, after what you did

**Comment by scarletbright** [comment]
> * `[2024-03-26 18:13:41 actix-rt|system:0|arbiter:1 ERROR] stream error: request parse error: invalid Header provided.` need more info, like version, more logs, after what you did

I just ran the install script, opened the port with ufw (on Ubuntu 20.04.6 LTS server), then if i'm trying to connect with my local computer (Mac), the Ubuntu server stream that error (quoted)

the client log:

![Screenshot 2024-03-27 at 08 38 01](https://github.com/ZingerLittleBee/server_bee-backend/assets/65666364/73077a5d-4d4f-4e9a-b4a7-43202383cb58)

**Comment by scarletbright** [comment]
> > > * `[2024-03-26 18:13:41 actix-rt|system:0|arbiter:1 ERROR] stream error: request parse error: invalid Header provided.` need more info, like version, more logs, after what you did
> > 
> > 
> > I just ran the install script, opened the port with ufw (on Ubuntu 20.04.6 LTS server), then if i'm trying to connect with my local computer (Mac), the Ubuntu server stream that error (quoted)
> > the client log:
> > ![Screenshot 2024-03-27 at 08 38 01](https://private-user-images.githubusercontent.com/65666364/317248382-73077a5d-4d4f-4e9a-b4a7-43202383cb58.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE1NDY0ODMsIm5iZiI6MTcxMTU0NjE4MywicGF0aCI6Ii82NTY2NjM2NC8zMTcyNDgzODItNzMwNzdhNWQtNGQ0Zi00ZTlhLWI0YTctNDMyMDIzODNjYjU4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI3VDEzMjk0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA5NTEzNGFhNmJmZmUwYjRmZjE1NzJiMDhhM2JmYzJmYmUwODg5NzYyNTRhMTdhM2Y4YTgyOWU2NDBiYzU4NjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.qGbz1xICgS18FQw7ugr0LCGarSsOxR8NPxQD73eRXLY)
> 
> I have never see this error before. Did you download the latest version？

yep, like this: ```bash <(curl -s https://raw.githubusercontent.com/ZingerLittleBee/server_bee-backend/main/script/startup.sh)```
